File: mobile/src/test/java/com/zacharytamas/often/models/managers/HabitManagerTest.java
package com.zacharytamas.often.models.managers;

import com.zacharytamas.often.models.Habit;

import org.junit.Test;

/**
 * Created by zacharytamas on 10/26/15.
 */
public class HabitManagerTest {

    private Habit createHabit() {
        Habit habit = new Habit();
        return habit;
    }

    @Test
    public void test_habitCompletion() {
        Habit habit = createHabit();
    }

}

File: mobile/src/test/java/com/zacharytamas/often/models/managers/HabitManagerTest.kt
package com.zacharytamas.often.models.managers

import com.zacharytamas.often.models.Habit
import junit.framework.Assert
import org.junit.Test

/**
 * Created by zacharytamas on 10/15/15.
 */
class HabitManagerTest {

    fun createHabit(): Habit {
        val habit = Habit();

        return habit
    }

    @Test
    fun test_habitCompletion() {

        val habit = this.createHabit();

//        Assert.assertTrue(false)
    }

}